RELEASES.md

```diff
@@ -1,3 +1,14 @@
+Version 1.29.2 (2018-10-11)
+===========================
+
+Compatibility Notes
+-------------------
+
+- [Fixed unsoundness in the aliasing rules, which caused miscompilation.][54639]
```

Comment:
I'd suggest something like "Work around aliasing-related LLVM bug, which caused miscompilations" here. The current description sounds to me like there was some kind of unsoundness in *Rust's* aliasing rules.